packages/exampleReactNative/App.tsx

```diff
@@ -0,0 +1,158 @@
+/**
```

Comment:
Can we add the same copyright headers as elsewhere in the project? (Just consistency)